### PR TITLE
[16.0][MIG] maintenance_sign_oca: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @pedrobaeza and @pilarvargas-tecnativa can you review it?

@Tecnativa

### CODE REVIEW

**Review by pilarvargas-tecnativa — APPROVED** [review]

File: maintenance_sign_oca/demo/sign_oca_role.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <odoo>
     <record id="role_maintenance_equipment_owner" model="sign.oca.role">
         <field name="name">Equipment owner</field>
-        <field name="partner_type">expression</field>
+        <field name="partner_selection_policy">expression</field>
```
Comment (pedrobaeza) [inline maintenance_sign_oca/demo/sign_oca_role.xml:4]: This highlights a migration script needed on the base module, isn't it?

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

File: maintenance_sign_oca/demo/sign_oca_role.xml (hunk 1 above)
Comment (victoralmau) [reply maintenance_sign_oca/demo/sign_oca_role.xml:4]: Exactly, already added https://github.com/OCA/sign/blob/16.0/sign_oca/migrations/16.0.1.1.1/pre-migration.py

**Pull request merged**